examples/tags-per-resource/README.md

```diff
@@ -0,0 +1,84 @@
+# Complete RDS example for PostgreSQL
```

Comment:
thanks for the PR @ruben0626 - we can just add your tag overrides to the `complete-mysql` and `complete-postgresql` instead of creating a new example project just for this. please add them to the first module instantiation of each (not the `db_default` one as this signifies what is provided by default)
- https://github.com/terraform-aws-modules/terraform-aws-rds/blob/0060e609b6547d538af11038d39a5ef508cefecd/examples/complete-mysql/main.tf#L61
- https://github.com/terraform-aws-modules/terraform-aws-rds/blob/0060e609b6547d538af11038d39a5ef508cefecd/examples/complete-postgres/main.tf#L61